app/views/teaching_events/about_git_events.html.erb

```diff
@@ -28,11 +28,20 @@
       <li>find out how to apply for teacher training</li>
       <li>explore what funding is available</li>
     </ul>
+
+<p>Watch our video to see how guests at a Get Into Teaching event have found help and advice on training to teach.</p>
```

Comment:
```suggestion
<p>Watch our video to see how people like you get help and advice on training to teach at Get Into Teaching events.</p>
```
This looks better now but is there another word we could use than `guests` - sounds a bit formal? NB I'm not sure about `people like you` but going with the second person tone of the page/social proof feel - happy for you to change!